financialaid/models_test.py

```diff
@@ -90,3 +87,25 @@ def test_save_and_log(self):  # pylint: disable=no-self-use
                 self.assertAlmostEqual(value, original_after_json[field])
             else:
                 assert value == original_after_json[field]
+
+    def test_to_dict(self):
+        """
+        Tests that to_dict() function on FinancialAid
+        """
+        financial_aid = FinancialAidFactory.create()
+        financial_aid_dict = financial_aid.to_dict()
```

Comment:
you may want to assert that all the keys you want are in the dict before the following asserts